samples/openapi3/client/petstore/python/docs/FakeApi.md

```diff
@@ -609,11 +609,7 @@ configuration = petstore_api.Configuration(
 with petstore_api.ApiClient() as api_client:
     # Create an instance of the API class
     api_instance = fake_api.FakeApi(api_client)
-    mammal = Mammal(
-        has_baleen=True,
-        has_teeth=True,
-        class_name="whale",
-    ) # Mammal | Input mammal
+    mammal = Mammal(None) # Mammal | Input mammal
```

Comment:
Looks like a regression here, no?

Reply:
not exactly. Because mammal is type any type, None is a valid selection based upon the schema being anyType. That payload would fail whale or zebra validation so this is highlighting how our python tooling needs to be smarter in how it handles composed anyType schema examples.